### ISSUE TITLE
Learning phase 0 and 1.

### ISSUE THREAD

**Issue by philipperemy** [philipperemy/keras-visualize-activations#2]
Thanks a lot for sharing!
One simple issue,

`Learning phase. 1 = Test mode (no dropout or batch normalization)`

However, I found in Keras (2.0.2)
` The learning phase flag is a bool tensor (0 = test, 1 = train)
`
And in practice I set the flag to 0 will get same output for same samples.

**Comment by philipperemy** [comment]
Fixed in https://github.com/philipperemy/keras-visualize-activations/pull/3
